components/config-mgmt-service/backend/postgres/policy_cookbooks.go

```diff
@@ -0,0 +1,82 @@
+package postgres
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/chef/automate/components/config-mgmt-service/backend"
+	"github.com/chef/automate/components/config-mgmt-service/errors"
+)
+
+func (p *Postgres) AddPolicyCookbooks(ctx context.Context,
```

Comment:
This file adds and retrieves policy file cookbook locks from postgresql. Before this information was pulled from the elasticsearch actions-* indexes.